app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,8 @@
     <string name="enter_your_greenhouse_id">Enter your greenhouse ID</string>
     <string name="greenhouse_id">Greenhouse ID</string>
     <string name="save">Save</string>
-
+    <string name="temperature">Temperature</string>
```

Comment:
Could use better names for string resources

Reply:
Okay, I will look into the strings and sure if Maggie is okay with it then I will be happy since idk much about design :D